### PR TITLE
minor tweaks from inkit

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes
minor tweaks from inkit

### Merge timing
asap

### CODE REVIEW

File: src/connections/destinations/catalog/inkit/index.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ title: Inkit Destination
 rewrite: true
 id: 5f0746ced1c79b49ddee49fd
 ---
-[Inkit](https://inkit.com){:target="_blank"} empowers organizations to securely generate and distribute documents - both digitally as well as through direct mail. 
-For example, automatically create and send electronic documents like invoices, reports, notices, and more using a magic link or e-delivery. Or generate and send documents for e-signature, storage, postcards, letters, and more, all powered by the Inkit integration for Segment.
+[Inkit](https://inkit.com){:target="_blank"} and Segment empower organizations to securely generate and distribute documents - both digitally as well as via direct mail.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/inkit/index.md:6]: @tcgilbert can we not use "via" and use "through"? "Via" isn't a word we use for the docs site as part of our style guide

File: src/connections/destinations/catalog/inkit/index.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ title: Inkit Destination
 rewrite: true
 id: 5f0746ced1c79b49ddee49fd
 ---
-[Inkit](https://inkit.com){:target="_blank"} empowers organizations to securely generate and distribute documents - both digitally as well as through direct mail. 
-For example, automatically create and send electronic documents like invoices, reports, notices, and more using a magic link or e-delivery. Or generate and send documents for e-signature, storage, postcards, letters, and more, all powered by the Inkit integration for Segment.
+[Inkit](https://inkit.com){:target="_blank"} and Segment empower organizations to securely generate and distribute documents - both digitally as well as via direct mail. 
+For example, automatically create and send electronic documents like invoices, reports, notices, and more via a magic link or e-delivery. Or generate and send documents for e-signature, storage, postcards, letters, and more, all powered by the Inkit integration for Segment.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/inkit/index.md:7]: @tcgilbert same here with the use of "via". "Via" isn't a word we use for the docs site as part of our style guide

Comment (tcgilbert) [reply src/connections/destinations/catalog/inkit/index.md]: yes! just updated it in last commit

File: src/connections/destinations/catalog/inkit/index.md (hunk 1 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/inkit/index.md]: yes! just updated it in last commit

**Comment by tcgilbert** [comment]
@stayseesong can we get this one included in Tuesday's deploy? Thanks!

**Review by stayseesong — APPROVED** [review]

**Pull request merged**